schemas/context/webreferrer.schema.json

```diff
@@ -28,15 +28,19 @@
                 "search_engine",
                 "typed_bookmarked",
                 "email",
-                "social"
+                "social",
+                "unknown",
+                "usenet"
               ],
           "meta:enum": {
             "internal": "Referral originated from the current domain or site.",
             "external": "Referral originated from a different domain or site that was not a search or social type.",
-            "searchEngine": "Search engine referrers are identified when visitors use a search engine to access the site.",
-            "typedBookmarked": "Typed/bookmarked referrers are identified when visitors type your site’s URL directly into their browser, or if they access the site by selecting bookmarks.",
+            "search_engine": "Search engine referrers are identified when visitors use a search engine to access the site.",
+            "typed_bookmarked": "Typed/bookmarked referrers are identified when visitors type your site’s URL directly into their browser, or if they access the site by selecting bookmarks.",
             "email": "A referring domain is considered as an email referring domain type when visitors click an emailed message link containing the protocol imap:// or mail:// and arrive at the site.",
-            "social": "The referrer is identified as a social network."
+            "social": "The referrer is identified as a social network.",
+            "unknown": "The referrer is unidentifiable (unknown). `No JavaScript` could be the cause.",
+            "usenet": "The URL of a referrer started with news://. As such, the referrer link was posted on a Usenet newsgroup rather than a web page."
```

Comment:
usenet should instead be NNTP or news.  Usenet is/was the name of a whole network, not just one specific usecase.